app/controllers/users/sessions_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Users::SessionsController < Devise::SessionsController
+  def after_sign_in_path_for(resource)
+    if resource.developer.present?
+      developer_path resource.developer
+    else
+      root_path
+    end
```

Comment:
You could do something like this if exit clauses are preferred over if/else's:

```suggestion
    return root_path unless resource.developer

    developer_path resource.developer

Reply:
This is actually nicer than mine. Committing it. Thanks for the suggestion.